optuna/importance/_fanova.py

```diff
@@ -0,0 +1,112 @@
+from collections import OrderedDict
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import CategoricalDistribution
+from optuna.distributions import DiscreteUniformDistribution
+from optuna.distributions import IntUniformDistribution
+from optuna.distributions import LogUniformDistribution
+from optuna.distributions import UniformDistribution
+from optuna.importance._base import BaseImportanceEvaluator
+from optuna.importance._base import get_distributions
+from optuna.importance._base import get_study_data
+from optuna.study import Study
+
+try:
+    from ConfigSpace import ConfigurationSpace
+    from ConfigSpace.hyperparameters import CategoricalHyperparameter
+    from ConfigSpace.hyperparameters import Hyperparameter
+    from ConfigSpace.hyperparameters import UniformFloatHyperparameter
+    from ConfigSpace.hyperparameters import UniformIntegerHyperparameter
+    from fanova import fANOVA
+
+    _available = True
+except ImportError as e:
+    ConfigurationSpace = None
+    Hyperparameter = None
+    fANOVA = None
+
+    _import_error = e
+    _available = False
+
+
+class FanovaImportanceEvaluator(BaseImportanceEvaluator):
+    def __init__(self) -> None:
```

Comment:
Users probably want to specify arguments passed to `fANOVA.__init__` such as the random seed `seed`. Should we support all arguments in this PR?